### DISCUSSION TITLE
HA integration showing way less usage than Bright App or SMETS2 Smart Meter

### DISCUSSION THREAD

**Discussion by cokecan72** [HandyHat/ha-hildebrandglow-dcc#310] [General]
Hello All

I am signed up to the Glow app so I can use this HA integration to pull in my gas/electricity usage and spend. It's been working really well up until recently when I noticed that the data visible in HA is either very low or non-existent compared to the Glow app and my actual SMETS2 smart meter. When I say low,  I mean like barely half of my actual usage or reporting as less than 1 kWh which is obviously just wrong. The usage in Glow and on my SMETS2 smart meter are the same so can only assume the issue is with my HA setup or integration? To clarify, I'm seeing the same behaviour with both gas and electricity usage. Interestingly the cost shown in HA is also reflective of the low usage seen in the integration and not based on the actual usage visible in the Glow app.

I have removed and re-added the latest version of the Hildebrand Glow (DCC) integration and re-auth'd my Glow account but it doesn't seem to have made any difference. Besides this there have been no changes bar HA updates applied to the system for some time. While I'm not sure when this issue occurred, I'm certain it was within the last 4-6 weeks.

I'm using v1.0.3 of Hildebrand Glow (DCC) installed via HACs and Home Assistant version 2023.2.4.

Can anyone offer and suggestions as to wants happening here or what I can do to try and diagnose and fix the problem?

Many thanks in advance.

Not sure if this helps, but I have attached some screenshots showing the discrepancy
![IMG_3327](https://user-images.githubusercontent.com/109978326/219057494-4959d2e4-f8be-424b-b481-bcaa692e6893.PNG)
![IMG_3328](https://user-images.githubusercontent.com/109978326/219057508-c434087d-89ae-4df6-a1b2-b3d5f376b187.PNG)
![IMG_3329](https://user-images.githubusercontent.com/109978326/219057525-8206fe13-102c-4856-b766-942b525dec36.PNG)
![IMG_3330](https://user-images.githubusercontent.com/109978326/219057535-b39fee67-99dc-4ec0-9bac-0ebe054d9312.PNG)

**Comment by HandyHat** [comment]
Hey @cokecan72, this sounds like it is the same issue as #305. Feel free to track that issue for updates!

**Reply by cokecan72** [reply]
Thank you for the reply HandyHat. I hadn't seen that bug, but you are 100% right. Seems like the exact issue I'm seeing.

I'll post in there and follow the bug to resolution
